Formula/babelfish.rb

```diff
@@ -24,6 +24,23 @@ def install
         gcflags: "all=-l -B -wb=false",
       )
     doc.install "README.md"
+    fish_function.install "babel.fish"
+  end
+
+  def caveats
+    <<~CAVEATS
+      The shell hook has been installed to which you can use by running:
```

Comment:
```suggestion
      The shell hook has been installed, you can use it by running:
```

Reply:
Thx apparently I was pretty tired when I wrote this haha